asf_floorplan_interpreter/pipeline/prodigy_to_yolo.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+# Convert json Prodigy output to Yolo8 input
+
+
+def scale_points_by_hw(lst, width, height):
+    "helper function to scale the x,y coordinates by object height and width"
+    return [[elem[0] / width, elem[1] / height] for elem in lst]
+
+
+def convert_prod_to_yolo(input_file_path, object_to_class_dict, output_file_path):
+    """
+    Takes a single prodigy label (dictionary, saved in json file format) and
```

Comment:
since the input file will contain labels for many floorplans at once, does it make sense that this function inputs the spans from a floorplan, and another function /script does the reading and saving of the data?

Reply:
ie..
```
data = read_jsonl(input_file_path)
spans_converted = []
for floorplan in data:
   spans_converted.append(convert_prod_to_yolo(floorplan, object_to_class_dict))

save_data(data, output_file_path)
```